ports/atmel-samd/boards/ugame10/mpconfigboard.h

```diff
@@ -47,3 +47,9 @@
 #define EXTERNAL_FLASH_DEVICES S25FL216K
 
 #include "external_flash/external_flash.h"
+
+#undef EXTRA_BUILTIN_MODULES
```

Comment:
Why is this undef needed? My guess is there is somewhere we're including `mpconfigboard.h` directly when we should use `py/mpconfig.h` instead